include/mruby.h

```diff
@@ -176,6 +176,8 @@ struct mrb_cache_entry {
 };
 #endif
 
+typedef uint32_t khint_t;
+
 struct mrb_jmpbuf;
```

Comment:
I don't like this disclosure. The scope should be as small as possible.
Can you elaborate?

Reply:
Fixed.